src/components/chat/MessageForm.js

```diff
@@ -90,6 +109,9 @@ class MessageForm extends Component {
                 value={message}
                 ref={(input) => { this.messageInput = input; }}
                 placeholder="Enter message" required></textarea>
+                <Button onClick={this.onSendMessage}>
+                  <FaArrowCircleRight size={30} />
+                </Button>
```

Comment:
@elimisteve I honestly don't remember whether it was there or not before, lol. 
But it does provide a bigger clicking scope and some default `react-bootstrap` styling.